website/content/middleware/cors.md

```diff
@@ -35,34 +35,41 @@ CORSConfig struct {
 
   // AllowOrigin defines a list of origins that may access the resource.
   // Optional. Default value []string{"*"}.
-  AllowOrigins []string `json:"allow_origins"`
+  AllowOrigins []string `yaml:"allow_origins"`
```

Comment:
Why `yaml` instead of `json`?

I'd prefer going with JSON as a default. Not a bid deal.

Reply:
It changed to yaml because I've copy-pasted the struct from the master branch (for consistency): https://github.com/labstack/echo/blob/master/middleware/cors.go#L14